subprojects/core/src/main/java/org/gradle/api/internal/file/archive/TarFileTree.java

```diff
@@ -43,6 +43,8 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public class TarFileTree extends AbstractArchiveFileTree {
+    public static final String TAR_ENTRY_PREFIX = "tar entry";
```

Comment:
Extracting these constants turned out to not be necessary, but I left them as a cleanup.